pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
+        Parameters
+        ----------
+        rule : str
+            The offset string or object representing target conversion.
+        *args
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
+        **kwargs
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
```

Comment:
I think it'd be good to do some research on what `get_resampler_for_grouping` does with them, which can be used, and what's the impact on this method.